fem/coefficient.hpp

```diff
@@ -946,6 +946,58 @@ class OuterProductCoefficient : public MatrixCoefficient
                      const IntegrationPoint &ip);
 };
 
+class QuadratureFunction;
+
+/// Vector quadrature function coefficient which requires that the quadrature rules used for this
```

Comment:
Please limit comments to 80 symbols per line, as in all other files.

Reply:
I'm surprised the ```make style``` doesn't fix such things. It should be fixed now.